### DISCUSSION TITLE
Using external packages when writing hooks

### DISCUSSION THREAD

**Discussion by txstc55** [pockethost/pockethost#288] [Q&A]
I have not used the hook yet, but I want to ask this question before diving into it:
If I have another node package that I want to use when defining hook, is there a way to add node package?
For example, if I have an external package that just gets the current weather, and i want to call it before updating my database, is there a way to add this package to the ftp server?

Another dumb question, if I just want to make a https request to another server, can I just add that to the hook?

**Comment by benallfree** [comment]
@txstc55 It depends on what PocketBase supports in goja. Since PocketHost doesn’t handle this, please start a discussion in the PocketBase forum and link to it here so we can follow.

**Reply by txstc55** [reply]
[added to the discussion](https://github.com/pocketbase/pocketbase/discussions/3332)

**Comment by ajomuch92** [comment]
@txstc55 Maybe you can try to use import maps as workaround

**Comment by benallfree** [comment]
@ajomuch92 @txstc55 I might be misunderstanding, wouldn't you be using a bundler like bun.sh anyway?
